workspaces/ui/src/components/diff/v2/DiffReviewExpanded.js

```diff
@@ -152,7 +152,7 @@ export const DiffReviewExpanded = (props) => {
   const { interaction, interactionScala } = currentInteraction;
 
   const { method, path } = interactionScala.request;
-  // track('Display Diff in Behavior Page', props);
+  // track('Display IAsyncTask in Behavior Page', props);
```

Comment:
Another bit of rename that you might want to revert as well, in case this comment is ever re-enabled!